src/protocol/http/src/main/java/org/apache/jmeter/protocol/http/proxy/DefaultSamplerCreator.java

```diff
@@ -283,32 +285,58 @@ public boolean isErrorDetected() {
      * @param sampler {@link HTTPSamplerBase}
      * @param request {@link HttpRequestHdr}
      */
-    protected void computeSamplerName(HTTPSamplerBase sampler,
-            HttpRequestHdr request) {
-        String prefix = request.getPrefix();
+    protected void computeSamplerName(HTTPSamplerBase sampler, HttpRequestHdr request) {
+        String prefix = request.getPrefix(); // ppp
```

Comment:
What is the meaning of *ppp* ?

Reply:
ppp is "ppp"refix name" i use this string to test (look at the image for the description of the modification and the test.
nnn is "nnn"ame i use this string to test the naming request.